HistContext/README.md

```diff
@@ -1,9 +1,14 @@
 ### Histtory Context gramplet
 
+![](./gramplet.png) 
 Gramplet to display historical events with the year they happened, and optionally the year  they ended.
+
 A double-click on a row will open your browser at the provided Link
 
-The gramplet works on the person and the relations view.
+The gramplet works on the person and the relations view and will us the bith and death of the active person.
```

Comment:
relations  = "Relationships Category view"
bith = birth

Reply:
Changed person to People, us to use and relations to Relationship Category
changed birt to birth
Committed and pushed